Automattic/wp-calypso
# Updating monitor settings property

Update the monitor_notify_users_emails to monitor_user_emails to match the endpoint response

#### Proposed Changes

* This PR is intended to reflect a change in the API response and updates the notification users email property for monitor from `monitor_notify_users_emails` to `monitor_user_emails`

#### Testing Instructions



* Run `git checkout update/agency-dashboard-update-notification-user-property`
* Run `yarn-start-jetpack-cloud`
* Visit the dashboard at `http://jetpack.cloud.localhost:3000/dashboard`
* Click on the clock icon
* Verify that the users email address is displayed when email notifications are enabled

#### Pre-merge Checklist





Related to #
